main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+func main() {
+	e := echo.New()
+	e.GET("/", func(c echo.Context) error {
+		return c.String(http.StatusOK, "Hello World!!")
+	})
+	e.Logger.Fatal(e.Start(":3000"))
```

Comment:
echoのドキュメントには乗っ取ってるし、goの文化的ではあるかもしれないけどこういう変数命名は良くないかなと思う

```suggestion
	app := echo.New()
	app.GET("/", func(c echo.Context) error {
		return c.String(http.StatusOK, "Hello World!!")
	})
	app.Logger.Fatal(e.Start(":3000"))
```